features/admin.branding.v1/api/layout.ts

```diff
@@ -52,12 +55,21 @@ export const useLayout = <Data = Blob, Error = RequestErrorInterface>(
         responseType: "blob",
         url:
             layout === PredefinedLayouts.CUSTOM
-                ? `${
-                    Config.getDeploymentConfig().extensions?.layoutStoreURL
-                        ? (Config.getDeploymentConfig().extensions.layoutStoreURL as string)
-                            .replace("${tenantDomain}", tenantDomain)
-                        : `https://${window.location.host}${basename}/libs/login-portal-layouts`}/body.html`
-                : `https://${window.location.host}${basename}/libs/login-portal-layouts/${layout}/body.html`
+                ? (
+                    brandingMode === BrandingModes.APPLICATION
```

Comment:
Instead of further nesting ternary expressions, let's extract this to a function with early returns.

eg:

```
function getLayoutURL(
    layout: string, 
    brandingMode: string, 
    appName: string, 
    tenantDomain: string, 
    basename: string
): string {
    const layoutStoreURL = Config.getDeploymentConfig().extensions?.layoutStoreURL as string;

    if (layout === PredefinedLayouts.CUSTOM) {
        const baseStoreURL = layoutStoreURL
            ? layoutStoreURL.replace("${tenantDomain}", tenantDomain)
            : `https://${window.location.host}${basename}/libs/login-portal-layouts`;

        if (brandingMode === BrandingModes.APPLICATION) {
            return `${baseStoreURL}/apps/${appName}/body.html`;
        }

        return `${baseStoreURL}/body.html`;
    }

    return `https://${window.location.host}${basename}/libs/login-portal-layouts/${layout}/body.html`;
}
```

Reply:
Will be fixed with a follow up PR.